proposals/OBv3p0/specification.md

```diff
@@ -0,0 +1,374 @@
+
+
+## Changes to the Data Model
+Several changes to the Open Badges 2.0 data model are proposed, in order to accomplish the alignment with Verifiable Credentials and the above goals:
+
+* _Align existing Open Badges VerificationObject properties with Verifiable Credential data model properties_: Verifiable Credentials have properties to describe an assertion of a claim and the instructions for cryptographically proving the claim. As seen in the examples below, some VC properties should replace the VerificationObject properties of Open Badges. For example, issuedOn will be replaced by [issuanceDate](https://www.w3.org/TR/vc-data-model/#issuance-date), expires by [expirationDate](https://www.w3.org/TR/vc-data-model/#expiration), and verification by [proof](https://www.w3.org/TR/vc-data-model/#proofs-signatures).
+
+* _Image_: The Open Badges assertion has an optional image property which is expected to contain the badge metadata baked into an image. The 2.0 specification references the image in the BadgeClass and historically it has been expected that the baked image uses the BadgeClass image. In this model, the baking should continue to be optional as it is less needed with VCs. However, it is critical to note that the VC community is discussing methods of embedding verifiable data into files such as images, QR codes, and PDFs. This may not be ready for 3.0 but could be ready for a future version and referenced to encourage piloting. The BadgeClass image is required in 2.0. In 3.0, it should be optional. The baking of the badge was a proposed method of transporting Open Badges. This is not required with VCs. Some badge issuers may wish to include images in their badges for when they are being displayed online and some wallets may wish to put the images to use. Open Badges 3.0 should continue support for images to enable new uses for them to be adopted in future releases.
+
+* _Add recipient to the credentialSubject.id_: VCs have credentialSubject which references the claim being being verified. The credentialSubject ID property can reference the subject of the credential. This property can contain a URI representing the subject which may be a URL or a DID. It should be discussed whether other profile properties could be optionally included in the credentialSubject.
+
+* _Issuer & Credential Subject Identities_: With VCs, [issuer](https://www.w3.org/TR/vc-data-model/#issuer) and [credentialSubject](https://www.w3.org/TR/vc-data-model/#credential-subject) id properties must be a URI. This could be an HTTP-based URL or a [Decentralized Identifier](https://www.w3.org/TR/did-use-cases/#intro).
+
+* _credentialSubject.hasCredential_: The hasCredential property from schema.org is intended to contain the BadgeClass properties including name, description, criteria, etc. hasCredential.id should contain the canonical url for the BadgeClass. hasCredential.type identifies the properties as belonging to the BadgeClass.
```

Comment:
What is the idea behind hasCredential? It's a bit confusing to me since the Badge and VC would now be "one" - wouldn't all of the 3.0 Badges have credentials?

Reply:
The Verifiable Credentials Data Model offers a mechanism for an issuer to make "claims" about a credential subject. The whole package is a "Verifiable Credential", and each of the claims form the "substance" of what the issuer is describing about the subject. In comparison to VCs that make claims about your birth date `"birthDate": "2000-01-01"`, educational claims are often about the "credentials" you have received. It is a bit of overloading of the term, and the selection of exactly which term to use to form this relationship is something that the IMS OB Workgroup will dive into if we decide to move forward with integrating OB and VC. In OB 2.0 the relevant term is `"badge"` which translates to English as "this assertion claims that the recipient *has* this badge, in that they have met its criteria, and it has been conferred to them". We could stick with that term, but this is an opportunity to discuss if `hasCredential` from schema.org or another more generic `hasAchievement` term etc would be more fitting.